{"sections":[],"schemaVersion":{"minor":3,"patch":0,"major":0},"variants":[{"paths":["\/documentation\/swifterrorkit\/sourceerrorattribute\/internal"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","metadata":{"modules":[{"name":"SwiftErrorKit"}],"symbolKind":"case","roleHeading":"Case","title":"SourceErrorAttribute.internal","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"`internal`","kind":"identifier"}],"role":"symbol","externalID":"s:13SwiftErrorKit06SourceB9AttributeO8internalyA2CmF"},"abstract":[{"text":"Error in internal application.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"Logic error, bug, or issue with internal operation, …"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"`internal`"}],"platforms":["macOS"]}]}],"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/internal"},"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute/internal":{"abstract":[{"text":"Error in internal application.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"Logic error, bug, or issue with internal operation, …"}],"type":"topic","title":"SourceErrorAttribute.internal","url":"\/documentation\/swifterrorkit\/sourceerrorattribute\/internal","role":"symbol","kind":"symbol","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/internal","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"`internal`","kind":"identifier"}]},"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","abstract":[{"type":"text","text":"Error attribution and commons."}],"type":"topic","title":"SwiftErrorKit","kind":"symbol","role":"collection","url":"\/documentation\/swifterrorkit"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute","kind":"symbol","abstract":[{"text":"An error attribute to include information about the high-level component","type":"text"},{"type":"text","text":" "},{"type":"text","text":"in which the error emerged."}],"role":"symbol","title":"SourceErrorAttribute","url":"\/documentation\/swifterrorkit\/sourceerrorattribute","navigatorTitle":[{"kind":"identifier","text":"SourceErrorAttribute"}],"type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"SourceErrorAttribute"}]}}}